scripts/android.py

```diff
@@ -0,0 +1,260 @@
+ 'kunall.gupta17@gmail.com'],
+tags = ['documentation', 'user guides'], # free text
+is_beginner = True,
+# 1: Coding, 2: User Interface, 3: Documentation & Training,
+# 4: Quality Assurance, 5: Outreach & Research
+categories = [3, 5],
+time_to_complete_in_days = 3, # must be between 3 and 7
+# Field currently not accessible via API. gci-support says it is coming soon.
+external_url = "https://github.com/zulip/zulip-gci/blob/master/tasks/android.md",
+private_metadata = "android-A",
+do_upload = args.force)
+
+
+
+
+upload_task(
+# https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf
+name = 'Write documentation for the Streams Drawer',
+description = """An overview descriptive text/Documentation and compiled like 
+https://zulip.readthedocs.io/ describing the streams drawer.
+
+Checklist:
+Unread counts (Currently there is a bug which is breaking the unread counts, 
+	but I hope it would be fixed soon) and it expands and shows the topics of 
+that stream. On clicking the Stream name you can directly narrow to that stream.""",
+status = 1, # 1: draft, 2: published
+max_instances = 1,
+mentors = ['niftynei@gmail.com', 'rishig@zulipchat.com', 'tabbott@zulipchat.com',
+ 'kunall.gupta17@gmail.com'],
+tags = ['documentation', 'user guides'], # free text
+is_beginner = True,
+# 1: Coding, 2: User Interface, 3: Documentation & Training,
+# 4: Quality Assurance, 5: Outreach & Research
+categories = [3, 5],
+time_to_complete_in_days = 3, # must be between 3 and 7
+# Field currently not accessible via API. gci-support says it is coming soon.
+external_url = "https://github.com/zulip/zulip-gci/blob/master/tasks/android.md",
+private_metadata = "android-A",
+do_upload = args.force)
+
+
+
+
+upload_task(
+# https://developers.google.com/open-source/gci/resources/downloads/TaskAPISpec.pdf
+name = 'Write documentation for the Streams Drawer',
+description = """An overview descriptive text/Documentation and compiled like 
+https://zulip.readthedocs.io/ describing the streams drawer.
+
+Checklist:
+Unread counts (Currently there is a bug which is breaking the unread counts,
+but I hope it would be fixed soon) and it expands and shows the topics of that 
+stream. On clicking the Stream name you can directly narrow to that stream.""",
+status = 1, # 1: draft, 2: published
+max_instances = 1,
+mentors = ['niftynei@gmail.com', 'rishig@zulipchat.com', 'tabbott@zulipchat.com',
+ 'kunall.gupta17@gmail.com'],
+tags = ['documentation', 'user guides'], # free text
+is_beginner = True,
+# 1: Coding, 2: User Interface, 3: Documentation & Training,
+# 4: Quality Assurance, 5: Outreach & Research
+categories = [3, 5],
+time_to_complete_in_days = 3, # must be between 3 and 7
+# Field currently not accessible via API. gci-support says it is coming soon.
+external_url = "https://github.com/zulip/zulip-gci/blob/master/tasks/android.md",
+private_metadata = "android-A",
+do_upload = args.force)
```

Comment:
this seems to be teh same as the previous one?

Also maybe we should add `android` to all the tags?